### ISSUE TITLE
How to integrate CppMicroservices with Qt (LIKE framework) to have something like TornadoFX or DromblerFX (OSGi) like

### ISSUE THREAD

**Issue by al-sabr** [CppMicroServices/CppMicroServices#505]
Greetings all!

I would like to know if someone already achieved integrating CppMicroservices with a UI framework like Qt?

I've come across those really advanced UI frameworks on Java OSGi side:

1. https://github.com/edvin/tornadofx
2. https://www.drombler.org/drombler-fx

And I've looked how easy they are to use and it seems that it is really well designed...

https://www.youtube.com/watch?v=liOFCH5MMKk

So basically you can just hot reaload/install any UI components without ending the OSGi runtime.

I was wondering if this was achieve with CppMicroservice and if you guys are already using something like this.

Best Regards.

**Comment by saschazelzer** [comment]
The framework is not tied to a particular use case, but I am not aware of anything similar to what you linked.

No one else commented so far, so I am closing this for now. Feel free to open a new issue if you have a more concrete question or problem.

**Comment by al-sabr** [comment]
@saschazelzer can you reopen this thread it seems that I found out what I was relating to for C++.

https://www.mitk.org/wiki/BlueBerry

BlueBerry is a UI Framework that is part of MITK project. But since MITK is a massive project I started an initiative and tried to move it one level lower within CTK Framework.

The repo is here : https://github.com/al-sabr/CTK/tree/ctk-blueberry-framework

Do you think you can help me to figure out the missing parts to make it work?

Best regards

**Comment by saschazelzer** [comment]
I am afraid this discussion doesn't belong here.

I don't have the capacity to maintain or work on BlueBerry and it is maintained by MITK developers currently. I doubt that CTK is a good place either, since none of the CTK-devs (AFAIK) know the BlueBerry codebase. There just doesn't seem to exist enough interest in the existing communities for this, so maintaining your own independent fork of it looks like the best way to go.
